extensions/amp-animation/0.1/web-animations.js

```diff
@@ -476,12 +496,19 @@ export class MeasureScanner extends Scanner {
 
   /**
    * @param {!WebAnimationDef|!Array<!WebAnimationDef>} spec
+   * @param {?JSONType=} opt_args
    * @return {!Promise<!Array<!InternalWebAnimationRequestDef>>}
    */
-  resolveRequests(spec) {
-    this.css_.withVars(this.vars_, () => {
-      this.scan(spec);
-    });
+  resolveRequests(spec, opt_args) {
```

Comment:
nit: might be a good idea to include a high-level overview of request resolution and animation lifecycle.